integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/spatial/SpatialWithinBoundingBoxSearchPredicateIT.java

```diff
@@ -127,7 +127,7 @@ public void unsupported_field_types() {
 	}
```

Comment:
I think you can squash all the `HSEARCH-3091 Test predicate-level boost with * predicate` commits together.